package.json

```diff
@@ -20,7 +20,7 @@
     "acorn": "^5.5.3",
     "acorn-es7": "^0.1.0",
     "acorn-jsx": "^4.1.1",
-    "acorn-object-rest-spread": "^1.1.0",
+    "acorn-stage3": "^0.6.0",
```

Comment:
If we set ecmaVersion to 8 or higher, rest is supported natively. 

Added stage3 to support dynamic imports.